oceanwaters/doc/setup_dev_env.md

```diff
@@ -20,12 +20,12 @@ The following minimum software set is required to build and run OceanWATERS
 (version numbers and identifiers are listed where specific versions are
 required):
 
--   The Ubuntu 18.04 (Bionic) Linux operating system, with:
+-   The Ubuntu 20.04 LTS (Focal Fossa) Linux operating system, with:
     -   The Bash command shell
     -   The git version control system
--   The Robot Operating System (ROS), Melodic Morenia distribution, with:
+-   The Robot Operating System (ROS), Noetic distribution, with:
```

Comment:
I don't like the comma after `(ROS),` seems like **ROS** is a thing and **Noetic** is another thing.
```suggestion
-   [ROS Noetic Ninjemys](http://wiki.ros.org/noetic) distribution, with:
```